src/webui/www/private/scripts/misc.js

```diff
@@ -76,6 +76,11 @@ function friendlyPercentage(value) {
     return percentage.toFixed(1) + "%";
 }
 
+function friendlyFloat(value, precision) {
+    var mult = Math.pow(10, precision);
+    return Math.round(value * mult) / mult;
```

Comment:
I fell over this... why multiply then divide?
can't you do `return Math.round(value).toFixed(2)`?

Reply:
This is so that we can round to a specific digit. Otherwise `Math.round(value)` will round to the nearest int, with `toFixed(2)` adding two zeros.

Reply:
Thanks for explaining it.
AFAIK javascript numbers are always `float` type, then this function can be just `return value.toFixed(precision);` right?

Reply:
> AFAIK javascript numbers are always float type, then this function can be just return value.toFixed(precision); right?

You're right. That's a simpler solution too.